django_dbq/models.py

```diff
@@ -69,9 +71,10 @@ class Job(models.Model):
     next_task = models.CharField(max_length=100, blank=True)
     workspace = JSONField(null=True)
     queue_name = models.CharField(max_length=20, default='default', db_index=True)
+    priority = models.PositiveSmallIntegerField(default=0)
 
     class Meta:
-        ordering = ['created']
+        ordering = ['priority', 'created']
```

Comment:
Should be `-priority`

Reply:
Fixed in https://github.com/dabapps/django-db-queue/pull/25/commits/c2c2a86b580a5218767791abddddc5ba0b6cd6e0